### DISCUSSION TITLE
How to catch one error from errors in react hook form

### DISCUSSION THREAD

**Discussion by thkim9012** [react-hook-form/react-hook-form#8074] [Q&A]
I'd like to catch only one error from errors,
The following code can catch one error, 
but it works strangely.... 
If the user ignores the first error and puts the focus on another error field, 
it focus first error field but alert show next error message.
please help....

![image](https://user-images.githubusercontent.com/36370258/159603132-dadd1937-3b60-4dbd-a6e6-4dd076fb3e18.png)

**Comment by Moshyfawn** [comment]
I've tried [re-creating you use-case](https://codesandbox.io/s/eloquent-satoshi-4r1ukr?file=/src/App.tsx) and, I think, it works fine. Can you possible look into my example and see if it's something matches your requirement?

**Reply by thkim9012** [reply]
Thank you, but the problem has not been solved...

I can reproduce it in your example..

Step1. Type "1234567890" in the first field, and click Submit button.

Step 2. Type 1234567891 in the second field, remove all text in the first field, and click the Submit button. (Do not click before removing the first field.)

Result: You can see the error message in the second field, but you can see the focus on the first field.
